dataclients/kubernetes/kube.go

```diff
@@ -1356,3 +1383,50 @@ func (c *Client) Close() {
 		close(c.quit)
 	}
 }
+
+func (c *Client) fetchDefaultFilterConfigs() map[resourceId]string {
+	if c.defaultFiltersDir == "" {
+		log.Debug("default filters are disabled")
+		return make(map[resourceId]string)
+	}
+
+	filters, err := c.getDefaultFilterConfigurations()
+
+	if err != nil {
+		log.WithError(err).Error("could not fetch default filter configurations")
+		return make(map[resourceId]string)
+	}
+
+	log.WithField("#configs", len(filters)).Debug("default filter configurations loaded")
+
+	return filters
+}
+
+const maxFileSize = 1024 * 1024 // 1MB
```

Comment:
please move the const to the top of the file

Reply:
Thinking about it, wouldn't be 16k enough.